packages/x-date-pickers-pro/src/DateRangePicker/DateRangePickerView.tsx

```diff
@@ -229,6 +229,7 @@ function DateRangePickerViewRaw<TInputDate, TDate>(
       disablePast,
       minDate,
       maxDate,
+      shouldDisableDate,
```

Comment:
This part of the code is very error-prone, maybe we could improve it later

Reply:
Do you have an idea about how to improve it? I only see two ways:

- Use providers to avoid having to propagate the information, and forget them somewhere
- make tests that loop on all the exported components to check we do not break props propagation

Reply:
Do you have an idea about how to improve it? I only see two ways:

- Use providers to avoid having to propagate the information, and forget them somewhere
- make tests that loop on all the exported components to check we do not break props propagation

Reply:
I did not think about it in depth tbh
We could indeed have a conformance test taking a component, 2-3 callbacks to interact with the DOM and which tests all the validation props (it would at least be a subset of our problem)